apiserver/backup.go

```diff
@@ -22,6 +22,7 @@ type backupHandler struct {
 	ctxt httpContext
 }
 
+// ServeHTTP implements http.Handler.
```

Comment:
Small nit. Would you consider changing this to `[http.Handler]` so go doc creates a link?